src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStep.java

```diff
@@ -31,7 +36,7 @@
  *
  * @author Kohsuke Kawaguchi
  */
-public class InputStep extends AbstractStepImpl implements Serializable {
+public class InputStep extends Step implements Serializable {
```

Comment:
Also not compatible in this case.

Reply:
(Part of `program.dat`. Also see https://github.com/jenkinsci/pipeline-plugin/commit/7a0f10c21ddc40dc20f62d1b9d27633a7bf8aed4)